docs/documentation/make-first-prototype/branching.md

```diff
@@ -0,0 +1,78 @@
+# Branching (showing pages depending on user input)
+
+Our first question asks the user how many balls they can juggle. We’re going to send them to an ‘ineligible’ page if they can only juggle 2 or less.
```

Comment:
```suggestion
Our first question asks the user how many balls they can juggle. We’re going to send them to an ‘ineligible’ page if they can only juggle 2 balls or less.
```

Reply:
Do we need to say what happens if they can juggle more than 2 balls?

Reply:
I guess its implied that what happened before (next question) will continue to happen.. but happy to make explicit if you'd like to

Reply:
No worries - that's a fair rationale. Thanks for explaining!